### PR TITLE
Add configuration option to prevent the transport using the RabbitMQ management API

### PR DESCRIPTION
**Pull request by abparticular**

### CODE REVIEW

**Comment by abparticular** [comment]
@TravisNickels there's some new test failures here - I plan to fix them before merging this.

**Comment by abparticular** [comment]
@TravisNickels this can be merged now, pending your review

**Review by TravisNickels — APPROVED** [review]

**Pull request merged**
